cnc_ctrl_v1/Report.cpp

```diff
@@ -236,7 +236,20 @@ void  returnPoz(){
     static unsigned long lastRan = millis();
     
     if (millis() - lastRan > POSITIONTIMEOUT){
+        char tmpbuf[80], buflen=0;
+        memset(tmpbuf, 0, sizeof(tmpbuf));
         
+        for (int i=0;i<3;i++){
+            if(axes[i].stalled > 5){
```

Comment:
why greater than 5?  It doesn't look like stalled can ever be anything by 0 or 1?

Reply:
Err... oops.  That's why I figured it should be a char (0-240 if report doesn't get called often. Fixed.